aws/ecs/README.md

```diff
@@ -0,0 +1,74 @@
+# CKEditor Collaboration Server On-Premise AWS ECS example
+
+Use this script to provision infrastructure with recommended settings and install CKEditor Collaboration Server on your AWS account.
+
+**Note**:
+This script uses AWS CDK. If you haven't worked with it before, we highly recommend to get familiar with it first:
+https://docs.aws.amazon.com/cdk/index.html
+
+## Parameters
+[Read more about AWS CDK context parameters](https://docs.aws.amazon.com/cdk/latest/guide/context.html)
+
+- `dockerRegistryAuthToken` - Authentication token to `docker.cke-cs.com` repository
+- `version` - Collaboration Server version (default `latest`)
+- `env` - A comma-separated list of CKEditor Collaboration Server configuration ([see available options](https://ckeditor.com/docs/cs/latest/onpremises/cs-onpremises/installation/docker.html#docker-container-environment-variables))
+
+## Quick start
+
+To use AWS CDK first You need an AWS account and AWS credentials. Read more about how to configure the environment [on official CDK docs](https://docs.aws.amazon.com/cdk/latest/guide/work-with.html).
+
+- install dependencies
```

Comment:
I would add that the commands should be executed on a developer's local computer

Reply:
Hmm, this is not always true. The script can be run on CI